website/docusaurus.config.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+module.exports = {
+  title: 'Watchman',
+  tagline: 'A file watching service',
+  url: 'https://facebook.github.io/watchman/',
+  baseUrl: '/',
```

Comment:
I needed to change this to `/watchman/` for the docs to render right; I'll fix this when I import to phabricato!

Reply:
Oh Thanks @wez , if you want that i change, let me know. I thought that was not necessary put `/watchman`,  is present in the url.

Reply:
It's ok; I've amended it into the phabricator diff.  Without this all the styling and js links are relative to `/` and the page looks unhappy.